#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint8 throttle_pot
uint8 direction
bool horn
bool park_brake_on
bool park_brake_off
bool speed_mode
bool torque_mode
uint8 max_speed
uint8 torque
uint8 pod_state_request
uint8 torque_limit
